### ISSUE TITLE
A Small Bug

### ISSUE THREAD

**Issue by lilstrawberry** [yxonic/DTransformer#3]
Hello! Thank you very much for your contribution. I found that there seems to be an extra for loop around lines 22-30 in the file "DTransformer/data.py" when "seq_len" is not set to None. To make it work, you can remove the "[" at line 22 and remove lines 29 and 30.

**Comment by yxonic** [comment]
thanks for reporting!

**Comment by TRRtrr** [comment]
Hello, I noticed while running this code that each dataset has only one training set and one testing set. However, it is mentioned in this paper that the final answer was obtained using a Five-fold Cross-validation approach. I would like to ask, do I need to manually cut out a validation set from the training set? Looking forward to receiving your guidance very much, thank you very much!
